index.bs

```diff
@@ -1819,6 +1820,24 @@ throughout the rest of this standard.
 
 A few abstract operations are used in this specification for utility purposes. We define them here.
 
+<h4 id="get-viewed-array-buffer" aoid="GetViewedArrayBuffer">GetViewedArrayBuffer ( O )</h4>
+
```

Comment:
Maybe add a link to https://bugs.ecmascript.org/show_bug.cgi?id=4369 in this first one. Something like

```
<div id="note">Ideally, this operation and its counterparts <a href="https://bugs.ecmascript.org/show_bug.cgi?id=4369">would be defined by ECMAScript</a>.</div>
```